sw/tools/generators/gen_airframe.ml

```diff
@@ -271,9 +271,10 @@ let preprocess_value = fun s v prefix ->
 let print_actuators_idx = fun out ->
   Hashtbl.iter (fun s (d, i) ->
     (* Set servo macro *)
-    fprintf out "#define Set_%s_Servo(_v) { \\\n" s;
-    fprintf out "  actuators[SERVO_%s_IDX] = Clip(_v, SERVO_%s_MIN, SERVO_%s_MAX); \\\n" s s s;
-    fprintf out "  Actuator%sSet(SERVO_%s_DRIVER_NO, actuators[SERVO_%s_IDX]); \\\n" d s s;
+    fprintf out "#define Set_%s_Servo(_pprzv,_s) { \\\n" s;
+    fprintf out "  actuators[SERVO_%s_IDX].pprz_val = ClipAbs( _pprzv, MAX_PPRZ); \\\n" s;
```

Comment:
Is there a need to do this here? Normally it should already be within this range, and clipping it can no longer influence the servo driver value.

Reply:
I was not sufficiently sure that all code everywhere did this properly, so I left this double clipping, hoping that the compiler would optimize it away if not needed.